src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -468,7 +468,7 @@ export class AppComponent implements OnInit, AfterViewChecked {
             case 'copyWithHeaders':
                 this.dataService.copyResults(event.selection, event.batchId, event.resultId, true);
                 break;
-            case 'copyHeaders':
+            case 'copAllHeaders':
```

Comment:
Still a typo here :P

Reply:
There must be something wrong with my keyboard aargh